shared/actions/team-building.tsx

```diff
@@ -8,21 +8,17 @@ import * as Saga from '../util/saga'
 import * as RPCTypes from '../constants/types/rpc-gen'
 import {TypedState} from '../constants/reducer'
 import {validateEmailAddress} from '../util/email-address'
-import flags from '../util/feature-flags'
 
-const closeTeamBuilding = (_: TypedState, {payload: {namespace}}: NSAction) => {
-  if (namespace === 'teams' && flags.teamsRedesign) {
```

Comment:
any flags that are off i deleted. trying to reduce anything thats already in or clearly out